test/lib/routes.js

```diff
@@ -96,5 +98,9 @@ module.exports = {
 
     "/script": function(write) {
         write(200, "<script src='/not/existent/file.js'></script><script>var foo = 'bar';</script><a href='/stage2'>stage2</a><script>var bar = 'foo';</script>");
+    },
+
+    "/encoded": function(write) {
+        write(200, iconv.encode("test ströng with ä few non english chåracters", "iso-8859-1"), "text/html; charset=ISO-8859-1");
```

Comment:
I'm wondering whether we should load in an external fixture file(s) with a few different encodings just to make sure that the text encoding of the file isn't resolved automatically by somebody's editor, etc.